### ISSUE TITLE
Definition of multi-task loss function

### ISSUE THREAD

**Issue by long123524** [lorenmt/mtan#45]
Hello, I would like to ask where is the definition of the multitasking loss function? Does the code lack the loss function call？

**Comment by lorenmt** [comment]
https://github.com/lorenmt/mtan/blob/268c5c1796bf972d1e0850dcf1e2f2e6598cc589/im2im_pred/utils.py#L14

**Comment by long123524** [comment]
你好，这个损失函数好像不是利用不确定性那篇文章的多任务损失函数方法，task_type=‘normal’这个是针对实例分割的嘛？

**Comment by lorenmt** [comment]
normal: 是指 normal prediction 不是 一个 task 的 definition 不是 multi-task weighing strategy。
你说的不确定性应该是 应该是 --weight uncert 定义在这里：https://github.com/lorenmt/mtan/blob/268c5c1796bf972d1e0850dcf1e2f2e6598cc589/im2im_pred/utils.py#L168

**Comment by long123524** [comment]
你好，我想问一下multi_task_trainer(train_loader, test_loader, multi_task_model, device, optimizer, scheduler, opt,
                           total_epoch=150):这个函数中opt是指定啥的呢？这个参数我并没有看懂，我运行的时候出现了以下错误，您能解答一下嘛？万分感谢
![image](https://user-images.githubusercontent.com/73996375/123885559-43b66400-d980-11eb-92a2-c477667a428b.png)

**Comment by lorenmt** [comment]
opt 是 argparse 的命名 方便训练的时候输入不同参数。 opt.temp 是 DWA 的 temperature 参数，具体参看 readme 的用法。
